setup.py

```diff
@@ -51,15 +51,12 @@
 
 
 if sys.platform != 'win32':
-    compile_args = dict( extra_compile_args=['-O2', '-march=native',
-                                             '-Wno-unused-function',
-                                             '-Wno-unneeded-internal-declaration',
-                                             '-Wno-#warnings'],
+    compile_args = dict( extra_compile_args=['-O2', '-march=native'],
                              extra_link_args=['-O2', '-march=native'])
+    libraries = ["m"]
 else:
-    compile_args = dict( extra_compile_args=['-Wunused-function',
-                                              '-Wunneeded-internal-declaration',
-                                              '-Wno-#warnings'])
+    compile_args = dict( extra_compile_args=[])
+    libraries = []
```

Comment:
Oh, that is weird. It means that ``acosh`` is not equal to ``ln( 1-sqrt(x**2 -1) )`` on MSVC. That's pretty strange. 

I agree, we should just merge this. directC doesn't work with the MSVC2008, which is fine, since hopefully not many people should be in this situation going forward.